tools/devpack.ps1

```diff
@@ -8,21 +8,26 @@
 # Specify --E2E to instead target the E2E test app.
 
 $rootPath = Split-Path -Parent $PSScriptRoot
-$project = "$rootPath\samples\FunctionApp\FunctionApp.csproj"
+$project = "$rootPath/samples/FunctionApp/FunctionApp.csproj"
```

Comment:
We should create a solution (sln) for the SDK with the analyzers, that would make this a bit simpler.

Reply:
Yes, it'd track a few other little nits too. Will follow up on those in the coming weeks.